src/Sulu/Bundle/MediaBundle/Media/Storage/LocalStorage.php

```diff
@@ -53,13 +48,16 @@ public function __construct($uploadPath, $segments, $logger = null)
     /**
      * {@inheritdoc}
      */
-    public function save($tempPath, $fileName, $version, $storageOption = null)
+    public function save($tempPath, $fileName, $preferredStorageOptions = null)
```

Comment:
Didn't know it was like this before... It could be in the same folder with a random sharding, but it's not very likely, and will end up in an even distribution no matter how many files or versions there are.